cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/AbstractNamespaceResourceDeleter.java

```diff
@@ -82,7 +82,10 @@
   @Override
   public void deleteResources(NamespaceMeta namespaceMeta) throws Exception {
     final NamespaceId namespaceId = namespaceMeta.getNamespaceId();
-
+    if (namespaceMeta.getNamespaceId().equals(NamespaceId.SYSTEM)) {
```

Comment:
We don't allow deletion of system namespace, right? Meaning no code path should even lead to this class?